packages/x-components/cypress.json

```diff
@@ -8,7 +8,7 @@
   "pluginsFile": "tests/plugins/index.js",
   "supportFile": "tests/support/index.ts",
   "integrationFolder": "tests/e2e",
-  "testFiles": ["cucumber/**/*.features"],
+  "testFiles": ["cucumber/**/*.feature"],
```

Comment:
is this change intended? I understood that we should stick to *.features because is faster on headless mode

Reply:
it should be a mistake

Reply:
Ups, I forgot to change back. 
Fixed now